### ISSUE TITLE
WCAG, Våre kontorer: miles-office-banner not clickable with keyboard

### ISSUE THREAD

**Issue by skaihigh** [miles-no/miles_web_components#69]
It is not possible to open the miles-office-banner with the keyboard.

- The button is not coded as a button and is not reachable
- The component should probably use the disclosure pattern to help screen readers: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/
- Make focus and hover styles similar to other places on the website.

![Image](https://user-images.githubusercontent.com/1680898/260924083-2e27d0e8-e0e0-4ca6-9dff-9044bc548bc2.png)
